bitnami/dataplatform-bp1/values.yaml

```diff
@@ -0,0 +1,233 @@
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    limits: {}
+    #  cpu: 250m
+    #  memory: 1Gi
+    requests:
+      cpu: 250m
+      memory: 256Mi
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: app.kubernetes.io/name
+            operator: In
+            values:
+            - zookeeper
+        topologyKey: "kubernetes.io/hostname"
+
+## Kafka Subchart parameters
+##
+kafka:
+  enabled: true
+  replicaCount: 3
+  resources:
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    limits: {} 
+    #  cpu: 250m
+    #  memory: 1Gi
+    requests: 
+      cpu: 250m
+      memory: 1Gi
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: app.kubernetes.io/name
+            operator: In
+            values:
+            - kafka
+        topologyKey: "kubernetes.io/hostname"
+    podAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+       - weight: 50
+         podAffinityTerm:
+           labelSelector:
+             matchExpressions:
+             - key: app.kubernetes.io/name
+               operator: In
+               values:
+                 - zookeeper
+           topologyKey: "kubernetes.io/hostname"  
+  zookeeper:
+    enabled: false
+  ## This value is only used when zookeeper.enabled is set to false
+  ##
+  externalZookeeper:
+    ## Server or list of external zookeeper servers to use.
+    ##
+    servers: dp-zookeeper
+
+## Spark Subchart parameters
+##
+spark:
+  enabled: true
+  ## Spark master specific configuration
+  ##
+  master:
+    resources:
+      # We usually recommend not to specify default resources and to leave this as a conscious
+      # choice for the user. This also increases chances charts run on environments with little
+      # resources, such as Minikube. If you do want to specify resources, uncomment the following
+      # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+      limits: {}
+      #  cpu: 250m
+      #  memory: 1Gi
+      requests:
+        cpu: 250m
+        memory: 1Gi
+    affinity:    
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchLabels:
+              app.kubernetes.io/name: spark
+              app.kubernetes.io/instance: dp-spark
```

Comment:
This seems to be depending on the release name too. Is it needed to set the affinity by default? Could you give us more context about it?

Reply:
I will change this.